CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/plugins/ExtractCoordsFromGraphPlugin.java

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2010-2022 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.mapview2.plugins;
+
+import au.gov.asd.tac.constellation.graph.GraphConstants;
+import au.gov.asd.tac.constellation.graph.GraphElementType;
+import au.gov.asd.tac.constellation.graph.GraphReadMethods;
+import au.gov.asd.tac.constellation.graph.schema.analytic.concept.SpatialConcept;
+import au.gov.asd.tac.constellation.graph.schema.visual.concept.VisualConcept;
+import au.gov.asd.tac.constellation.plugins.Plugin;
+import au.gov.asd.tac.constellation.plugins.PluginException;
+import au.gov.asd.tac.constellation.plugins.PluginInfo;
+import au.gov.asd.tac.constellation.plugins.PluginInteraction;
+import au.gov.asd.tac.constellation.plugins.PluginType;
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameters;
+import au.gov.asd.tac.constellation.plugins.templates.PluginTags;
+import au.gov.asd.tac.constellation.plugins.templates.SimpleReadPlugin;
+import au.gov.asd.tac.constellation.views.mapview2.MapViewTopComponent;
+import au.gov.asd.tac.constellation.views.mapview2.markers.PointMarker;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.openide.util.NbBundle;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ * Plugin to extract geo-coordinates form graph node
+ *
+ * @author altair1673
+ */
+@ServiceProvider(service = Plugin.class)
+@PluginInfo(pluginType = PluginType.SEARCH, tags = {PluginTags.SEARCH})
+@NbBundle.Messages("ExtractCoordsFromGraphPlugin=Extracts geographic coordinates from the graph")
+public class ExtractCoordsFromGraphPlugin extends SimpleReadPlugin {
+    private static final Logger LOGGER = Logger.getLogger(ExtractCoordsFromGraphPlugin.class.getName());
+
+    private static final double POINT_MARKER_X_OFFSET = 95;
+    private static final double POINT_MARKER_Y_OFFSET = 245;
+
+    private MapViewTopComponent mapViewTopComponent;
+
+    public ExtractCoordsFromGraphPlugin() {
+
+    }
```

Comment:
given there is another constructor for it, unless there is something else that can be added to this definition, I would remove this (seems like `mapViewTopComponent` needs to be instantiated to use the plugin anyway)